sphinx_gallery/backreferences.py

```diff
@@ -103,7 +103,12 @@ def identify_names(code):
     for name, full_name in finder.get_mapping():
         # name is as written in file (e.g. np.asarray)
         # full_name includes resolved import path (e.g. numpy.asarray)
-        module, attribute = full_name.rsplit('.', 1)
+        splitted = full_name.rsplit('.', 1)
+        if len(splitted) == 1:
+            module = splitted[0]
+            attribute = None
+        else:
+            module, attribute = splitted
```

Comment:
I would test for length 1(or not 2) and do `continue`. That way you don't save the module, or anything else that is not a `module, attribute` pair and you save yourself the need to have a check later on.